src/diffchecker.js

```diff
@@ -1,26 +1,45 @@
+const Diff = require('diff');
+
+function normalizeLines(str) {
+  const strWithNewLine = str.endsWith('\n') ? str : str + '\n';
+
+  return strWithNewLine.split('\n').map(line => line.replace(/\s+$/, '')).join('\n');
```

Comment:
This is to clear out trailing white spaces. The package has a property to ignore white spaces, but when trying to rebuild the diff file, it removed all the indentation.